pkg/ocm/clusterautoscaler.go

```diff
@@ -134,8 +143,17 @@ func (c *Client) CreateClusterAutoscaler(clusterId string, config *AutoscalerCon
 	return response.Body(), nil
 }
 
-func (c *Client) UpdateClusterAutoscaler(clusterId string, config *AutoscalerConfig) (*cmv1.ClusterAutoscaler, error) {
-	object, err := BuildClusterAutoscaler(config).Build()
+func (c *Client) UpdateClusterAutoscaler(clusterId string, isHostedCp bool,
```

Comment:
Is cluster id actually used here?

Reply:
@gdbranco Yup seems so, when it sends the API request it uses the cluster ID